### PR TITLE
Add Missing Type Annotations

### LINKED ISSUE #15: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_veml6075.py:70
- [ ] adafruit_veml6075.py:157
- [ ] adafruit_veml6075.py:165
- [ ] adafruit_veml6075.py:172

### PR DESCRIPTION
**Pull request by tcfranks**
resolves #15 
submitted for review/comment

### CODE REVIEW

File: adafruit_veml6075.py [hunk 1]
```diff
@@ -126,25 +132,25 @@ def _take_reading(self):
...
     @property
-    def uv_index(self):
+    def uv_index(self) -> float:
         """The calculated UV Index"""
         self._take_reading()
         return ((self._uvacalc * self._uvaresp) + (self._uvbcalc * self._uvbresp)) / 2
 
     @property
-    def integration_time(self):
+    def integration_time(self) -> Union[None, int]:
```
Comment (tekktrik) [inline adafruit_veml6075.py]: This can't return `None`, so it should just be `int`

File: adafruit_veml6075.py [hunk 2]
```diff
@@ -33,6 +33,12 @@
 from adafruit_bus_device.i2c_device import I2CDevice
 from micropython import const
 
+try:
+    from typing import Union
```
Comment (tekktrik) [inline adafruit_veml6075.py]: `Union` will be removed here, so add the pylint disable for unused import.

File: adafruit_veml6075.py [hunk 3]
```diff
@@ -96,7 +102,7 @@ def __init__(
         # read ID!
         veml_id = self._read_register(_REV_ID)
         if veml_id != 0x26:
-            raise RuntimeError("Incorrect VEML6075 ID 0x%02X" % veml_id)
+            raise RuntimeError(f"Incorrect VEML6075 ID {hex(veml_id)}")
```
Comment (tekktrik) [inline adafruit_veml6075.py]: I do like the uppercase, could you do:

```python
raise RuntimeError(f"Incorrect VEML6075 ID {hex(veml_id).upper()}")
```

**Review by tekktrik — CHANGES_REQUESTED** [review]
Thanks for the submission!  A few bits of feedback:

File: adafruit_veml6075.py (hunk 1 above)
Comment (tcfranks) [reply adafruit_veml6075.py]: oof.....why did I see that?

**Comment by tcfranks** [comment]
resubmitted for review of changes

File: adafruit_veml6075.py [hunk 4]
```diff
@@ -33,6 +33,12 @@
 from adafruit_bus_device.i2c_device import I2CDevice
 from micropython import const
 
+try:
+    from typing import Union  # pylint: disable=unused-import
```
Comment (tekktrik) [inline adafruit_veml6075.py]: `Union` isn't being used, so:

```suggestion
    import typing  # pylint: disable=unused-import
```

**Review by tekktrik — CHANGES_REQUESTED** [review]

**Review by tekktrik — APPROVED** [review]
LGTM!

**Pull request merged**